activerecord/lib/active_record/connection_adapters/abstract/transaction.rb

```diff
@@ -641,6 +648,12 @@ def rollback_transaction(transaction = nil)
       end
 
       def within_new_transaction(isolation: nil, joinable: true)
```

Comment:
```suggestion
      def within_new_transaction(isolation: nil, fixtures: false)
```

I think we should rename the keyword arg too.